libs/sdk-ui-dashboard/src/model/commands/drillTargets.ts

```diff
@@ -0,0 +1,43 @@
+// (C) 2021 GoodData Corporation
+
+import { ObjRef } from "@gooddata/sdk-model";
+import { IAvailableDrillTargets } from "@gooddata/sdk-ui";
+import { IDashboardCommand } from "./base";
+
+/**
+ * Add widget drill targets
+ *
+ * @internal
```

Comment:
All the `@internal` should be `@alpha` in this file after #1562

Reply:
Done